mtcpclient/calculate.go

```diff
@@ -39,7 +39,9 @@ func (gc GroupOfConnections) calculateMetricsReport(status tcpclient.ConnectionS
 			}
 		}
 	}
-	mr.avg = mr.total / time.Duration(mr.numberOfConnections)
+	if mr.numberOfConnections > 0 {
```

Comment:
I'd skip this change, the original code changed in the #50

Reply:
but I see that you need with the current code..., so let's clean this when we merge the other

Reply:
![Conflicts are coming](https://memegenerator.net/instance/46907994/winter-is-coming-brace-yourself-merge-conflicts-are-coming)